src/core/ipc_handler/unix_socket.rs

```diff
@@ -18,7 +18,7 @@ use std::io::{Error, ErrorKind};
 #[cfg(not(feature = "no-fs-permission-check"))]
 use std::os::unix::fs::MetadataExt;
 
-const DEFAULT_SOCKET_PATH: &str = "/tmp/parsec/parsec.sock";
+const DEFAULT_SOCKET_PATH: &str = "/run/parsec/parsec.sock";
```

Comment:
There is already a way to modify the path of the socket at runtime, so for test I think we can change this path to `/tmp/parsec.sock` in the `e2e_tests` folder directly.

Reply:
There is? Wow, I had some good foresight there